### PR TITLE
Update powershell_powerview_malicious_commandlets.yml - Addition and Miscellaneous

### PR DESCRIPTION
**Pull request by leegengyu**
Added new commandlet names based on aliases seen in https://github.com/PowerShellMafia/PowerSploit/blob/master/Recon/PowerView.ps1, fixed a typo, and improved formatting.

### CODE REVIEW

File: rules/windows/powershell/powershell_powerview_malicious_commandlets.yml [hunk 1]
```diff
@@ -1,96 +1,146 @@
 title: Malicious PowerView PowerShell Commandlets
 id: dcd74b95-3f36-4ed9-9598-0490951643aa
 status: experimental
-description: Detects Commandlet names from PowerView of PowerSploit exploitation framework
+description: Detects Commandlet names from PowerView of PowerSploit exploitation framework.
 date: 2021/05/18
```
Comment (frack113) [inline rules/windows/powershell/powershell_powerview_malicious_commandlets.yml:5]: As it is a detection modification, add
`modified: 2021/07/03`

File: rules/windows/powershell/powershell_powerview_malicious_commandlets.yml [hunk 2]
```diff
@@ -1,96 +1,146 @@
...
         - Find-LocalAdminAccess
+        - Invoke-EnumerateLocalAdmin
         - Find-DomainLocalGroupMember
+        - Get-NetDomainTrust
         - Get-DomainTrust
+        - Get-NetForestTrust
         - Get-ForestTrust
+        - Find-ForeignUser
         - Get-DomainForeignUser
+        = Find-ForeignGroup
```
Comment (frack113) [inline rules/windows/powershell/powershell_powerview_malicious_commandlets.yml:141]: should be `- Find-ForeignGroup`

**Review by frack113 — CHANGES_REQUESTED** [review]
Hi,
Just find 2 changes to make.
Thanks

**Comment by leegengyu** [comment]
Thank you @frack113 for pointing those 2 issues out and @Neo23x0 for fixing them!

**Pull request merged**